code/game/objects/items/devices/forcefieldprojector.dm

```diff
@@ -16,6 +16,8 @@
 	var/max_fields = 3
 	var/list/current_fields
 	var/field_distance_limit = 7
+	var/creation_time = 10
+	var/forceproj_busy = FALSE
```

Comment:
Also snakecase this

Reply:
```suggestion
	var/force_proj_busy = FALSE
```
You're welcome ;)

Reply:
Snakecasing done, appreciate the feedback!